packages/accordions/src/styled/StyledContent.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'steppers.step_content';
+
+export interface IStyledContent {
+  isActive?: boolean;
+}
+
+const inactiveStyling = css`
+  & > div {
```

Comment:
Rather than have a naked `<div>` within the `StepContent` would it make sense to have this a styled component? We could pass an `isActive` prop to it to control this animation.

Reply:
A styled component feels good. I created a `StyledInnerContent` and moved transition styles over to it.